src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Wasm/WasmBase.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Runtime.Intrinsics.Wasm
     [Intrinsic]
     internal abstract class WasmBase
     {
-        public static bool IsSupported { get; }
+        public static bool IsSupported { get => IsSupported; }
```

Comment:
can linker not detect the auto backing field (with `[CompilerGenerated]`) and drop it?

Reply:
This has nothing to do with the linker, all hardware intrinsics are written this way; it helps with accessing them from reflection (among other things).